crawl-ref/source/ouch.cc

```diff
@@ -796,6 +796,40 @@ static bool _is_damage_threatening (int damage_fraction_of_hp)
                 || random2(100) < hp_fraction);
 }
 
+// Palentongas curl up after the first time they've been hit in a round.
+static void _consider_curling(kill_method_type death_type)
+{
+    // can't curl
+    if (!you.has_mutation(MUT_CURL))
+        return;
+
+    // already curled
```

Comment:
Nit: I think these two comments are probably not needed; the code is clear enough.